tests/blob.rs

```diff
@@ -40,16 +48,19 @@ fn gif_blob_too_small_test() {
 fn blob_test_partial_ico() {
     let data = vec![
         // Header (says 6 images are included)
-        0x00, 0x00, 0x01, 0x00, 0x06, 0x00,
-        // Image 1 (16x32)
-        0x10, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-        // Image 2 (10x100)
-        0x0A, 0x64, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-        // Image 3 (255x? run out of bytes)
-        0xFF
+        0x00, 0x00, 0x01, 0x00, 0x06, 0x00, // Image 1 (16x32)
+        0x10, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+        0x00, // Image 2 (10x100)
+        0x0A, 0x64, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+        0x00, // Image 3 (255x? run out of bytes)
```

Comment:
Another instance where comments were intended for a specific layout get messed up